lib/drum.hoon

```diff
@@ -94,9 +107,17 @@
   ==                                                    ::
 ::
 ++  drum-port
-  |=  old/?(drum-part drum-part-old)  ^-  drum-part
+  |=  old/?(drum-part drum-part-old)
+  ^-  drum-part
   ?-  &2.old
-    $1  old
+    $2  old
+    $1  :+  %drum  %2
```

Comment:
By convention, this is usually `$(old old(+< %2, ...))` so that adding `%3` will be easier. Shouldn't be that difficult to change then tho.